test/integration/sagemaker/test_training.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2018-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+from __future__ import absolute_import
+
+import os
+
+import pytest
+from sagemaker.mxnet.estimator import MXNet
+from sagemaker import utils
+
+from test.integration import RESOURCE_PATH
+from timeout import timeout
+
+DATA_PATH = os.path.join(RESOURCE_PATH, 'mnist')
+SCRIPT_PATH = os.path.join(DATA_PATH, 'mnist.py')
+
+
+@pytest.mark.deployment_test
+def test_single_machine(sagemaker_session, ecr_image, instance_type):
+    _fit_estimator(1, instance_type, sagemaker_session, ecr_image)
```

Comment:
Optional - we can make the instance count a comma delimited parameter, something like this:

```
--instance-count 1,2
```

Reply:
done. I'll probably do something similar for Python version and processor in a later PR